### PR TITLE
Add release notes for 0.11.1

### PR DESCRIPTION
**Pull request by tgodzik**
We need to do another release to fix the performance issues.

### CODE REVIEW

**Review by ckipp01 — APPROVED** [review]
👍🏼 Good job getting the performance issues tackled so quickly everyone! 🚀

File: website/blog/2022-01-16-aluminium.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.1 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+Metals v0.11.1 is bugfix releas needed to fix a couple of performance
```
Comment (kpodsiad) [inline website/blog/2022-01-16-aluminium.md:8]:
```suggestion
Metals v0.11.1 is a bugfix release needed to fix a couple of performance
```

**Review by kpodsiad — APPROVED** [review]

File: website/blog/2022-01-16-aluminium.md (hunk 1 above)
Comment (tgodzik) [reply website/blog/2022-01-16-aluminium.md]: Good catch! Fixed.

**Comment by tgodzik** [comment]
If no one objects I am going to start the release in circa 30 minutes.

**Review by dos65 — APPROVED** [review]
:+1:

**Pull request merged**